Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -1676,6 +1677,17 @@ def prepare_PAT(self, sequence = "miniAOD"):
 
         return
 
+    def prepare_PATGEN(self, sequence = "miniAOD"):
+        ''' Enrich the schedule with PATGEN '''
+        self.loadDefaultOrSpecifiedCFF(sequence,self.PATDefaultCFF,1) #this is unscheduled
```

Comment:
something similar to ``` self.labelsToAssociate.append('patTask') ``` present in prepare_PAT should be present
and better point to something restricted to the miniGEN.

@Dr15Jones is the task association to the schedule essential to run a job?
I thought it was.

@perrozzi please clarify which cmsDriver command was used to test this 93X code.
Thank you.